.github/workflows/browser-tests.yml

```
@@ -0,0 +1,154 @@
+              run: echo "WEB_HOST=http://varnish" >> $GITHUB_ENV
+
+            - uses: actions/checkout@v2
+
+            - name: Setup PHP Action
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: 7.4
+                  coverage: none
+
+            - if: inputs.composer-package-version != ''
+              name: Set COMPOSER_ROOT_VERSION for packages with cyclic dependencies
+              run: echo "COMPOSER_ROOT_VERSION=${{ inputs.composer-package-version }}" >> $GITHUB_ENV
+
+            - if: env.SATIS_NETWORK_KEY != ''
+              name: Add composer keys for private packagist
+              run: |
+                  composer config http-basic.updates.ibexa.co $SATIS_NETWORK_KEY $SATIS_NETWORK_TOKEN
+                  composer config github-oauth.github.com $TRAVIS_GITHUB_TOKEN
+              env:
+                  SATIS_NETWORK_KEY: ${{ secrets.SATIS_NETWORK_KEY }}
+                  SATIS_NETWORK_TOKEN: ${{ secrets.SATIS_NETWORK_TOKEN }}
+                  TRAVIS_GITHUB_TOKEN: ${{ secrets.TRAVIS_GITHUB_TOKEN }}
+
+            - uses: "ramsey/composer-install@v1"
+              with:
+                  dependency-versions: "highest"
+                  composer-options: "--prefer-dist --no-progress"
+
+            - name: Set up whole project using the tested dependency
+              run: ./vendor/bin/prepare_project_edition.sh ${{ inputs.project-edition }} ${{ inputs.project-version }} ${{ inputs.setup }} ${{ inputs.php-image }}
+
+            - if: inputs.multirepository
+              name: Set up multirepository build
+              run: $HOME/build/project/vendor/ezsystems/behatbundle/bin/.travis/prepare_multirepository_setup.sh
+
+            - if: inputs.test-setup-phase-1 != ''
+              name: Run first phase of tests setup
+              run: |
+                  cd ${HOME}/build/project
+                  docker-compose --env-file=.env exec -T --user www-data app sh -c "vendor/bin/ezbehat ${{ inputs.test-setup-phase-1 }}"
+
+            - if: inputs.test-setup-phase-2 != ''
+              name: Run second phase of tests setup
+              run: |
+                  cd ${HOME}/build/project
+                  docker-compose --env-file=.env exec -T --user www-data app sh -c "composer run post-install-cmd"
+                  docker-compose --env-file=.env exec -T --user www-data app sh -c "vendor/bin/ezbehat ${{ inputs.test-setup-phase-2 }}"
+
+            - name: Run tests
+              run: |
+                  cd ${HOME}/build/project
+                  docker-compose --env-file=.env exec -T --user www-data app sh -c "vendor/bin/ezbehat ${{ inputs.test-suite }}"
+
+            - if: always()
+              name: Upload tests report
+              run: |
+                  cd ${HOME}/build/project
+                  vendor/bin/ezreport
+
+    report-results:
+        runs-on: ubuntu-latest
+        timeout-minutes: 3
+        if: always()
```

If I understand correctly the whole point of this job is to report when it's triggered from **not** a pull request.

So it should have been:
```suggestion
        if: always() && github.event_name != 'pull_request'
```
because of this paragraph about `always()`:
> You can use the following status check functions as expressions in if conditionals. A default status check of success() is applied unless you include one of these functions.

This would make the check for reporting step unnecessary.

---

You're 100% right, thanks! I've moved the condition so that it's applied for the whole `report-results` job.